pkg/typedef/typedef.go

```diff
@@ -197,32 +197,26 @@ const (
 	CacheArrayLen
 )
 
-func prettyCQL(query string, values Values, types Types) string {
-	if len(values) == 0 {
+func prettyCQL(query string, values Values, types []Type) string {
```

Comment:
As we discussed, it is better to make prettyCQL return error, so that lack of data could be handled properly isntead of paniciking.

Reply:
That's my next step, I wanted to simplify the code, remove the Go Iterators